default/stringtables/es.txt

```diff
@@ -5917,8 +5917,8 @@ Rango de detección
 #*ARMOR_TEXT
 #* <not translated 1970-01-01 01:00:00>
 
-#*SLOT_TITLE
-#* <not translated 1970-01-01 01:00:00>
+SLOT_TITLE
+Hueco
```

Comment:
I think I prefer "Ranura" rather than "Hueco".
Not quite happy with any of those, but _ranura_ has better support in dictionaries.

https://www.wordreference.com/es/en/translation.asp?spen=ranura (the one _para ensamble_).